packages/ui/src/__tests__/atoms/Button.test.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { fireEvent, render, screen } from '@testing-library/react'
+import { describe, expect, it, vi } from 'vitest'
+import { dimensions, colors } from '../../styles'
+import "@testing-library/jest-dom";
```

Comment:
Unnecessary import, it´s already in setup files, Please remove it

Reply:
Done, thanks